test/Microsoft.AspNet.Mvc.TagHelpers.Test/JavaScriptUtilityTest.cs

```diff
@@ -2,6 +2,7 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
+using Microsoft.AspNet.Mvc.TagHelpers.Internal;
 using Xunit;
 
 namespace Microsoft.AspNet.Mvc.TagHelpers.Test
```

Comment:
- namespace should be that of the class you're testing.
- class name = JavaScriptEncoderTest